src/models/src/NITableViewModel.h

```diff
@@ -21,7 +21,7 @@
 #import "NIPreprocessorMacros.h" /* for weak */
 
 #if NS_BLOCKS_AVAILABLE
-typedef UITableViewCell* (^NITableViewModelCellForIndexPathBlock)(UITableView* tableView, NSIndexPath* indexPath, id object);
+typedef UITableViewCell * _Nonnull (^NITableViewModelCellForIndexPathBlock)(UITableView* _Nonnull tableView, NSIndexPath* _Nonnull indexPath, _Nonnull id object);
```

Comment:
Not sure if it matters but I wonder if `id _Nonnull object` might be better. Doesn't seem like a blocker though.